Examples/Server/Sources/ServerExample/ServerExample.swift

```diff
@@ -0,0 +1,65 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OTel open source project
+//
+// Copyright (c) 2024 Moritz Lang and the Swift OTel project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Hummingbird
+import Logging
+import NIO
+import OTel
+import OTLPGRPC
+import ServiceLifecycle
+import Tracing
+
+@main
+enum ServerMiddlewareExample {
+    static func main() async throws {
+        // Bootstrap the logging backend with the OTel metadata provider which includes span IDs in logging messages.
+        LoggingSystem.bootstrap { label in
+            var handler = StreamLogHandler.standardError(label: label, metadataProvider: .otel)
+            handler.logLevel = .trace
+            return handler
+        }
+
+        // Configure OTel resource detection to apply attributes to automatically apply helpful attributes to events.
```

Comment:
```suggestion
        // Configure OTel resource detection to automatically apply helpful attributes to events.
```

Reply:
Oops; fixed.

Did I mention those attributes get applied? /s 🙃

Reply:
I'm glad you applied the fix 🤪